tests/scripts/hpo/hpo_multiple_exp_tests.sh

```diff
@@ -235,7 +235,9 @@ function hpo_multiple_exp_test() {
 
 			http_code=$(tail -n1 <<< "${get_trial_json}")
 			response=$(echo -e "${get_trial_json}" | tail -2 | head -1)
```

Comment:
It looks like this line can be moved into the `curl_check_error` function as well

Reply:
Moved